### ISSUE TITLE
A question about the accuracy reported

### ISSUE THREAD

**Issue by brandon9838** [XLechter/Detail-Preserved-Point-Cloud-Completion-via-SFA#1]
The reported numbers are Chamfer Distance multiplied by 10^4.
However, compared to the numbers in the original PCN paper, say 0.009720 for FC, I think it should be Chamfer Distance multiplied by 10^3.
Did I miss anything? Or the metric is different from PCN?
Thanks for replying!

**Comment by XLechter** [comment]
Hi@brandon9838 ,
Thank you for your commit first.
You are right if we use the metric standard in PCN.
But actually we follow the standard in TopNet which the CD loss is multiplied by 10^4.
We find the main reason caused this is due to the different data normalization scale from ShapeNet, so I think both standard are OK.
Note that in the code we use the dataset in PCN, so the results will be CD loss is multiplied by 10^3.
Hope this helps : )

**Comment by Leerw** [comment]
checkpoint provided not work, cannot produce any sensible point clouds

**Comment by XLechter** [comment]
@Leerw Which checkpoint you use? I have checked the checkpoint before I submit and confirm it can reproduce the results in the paper.

**Comment by XLechter** [comment]
I re-check the checkpoint and find it correct. For RFA, it produce the following results:
Average Chamfer distance: 0.008064
Chamfer distance per category
0.004760 &0.010163 &0.008627 &0.008557 &0.007036 &0.010514 &0.007352 &0.007499 &0.008063538116402924

I want to know if anyone meet the same problem with @Leerw ?

**Comment by Leerw** [comment]
No sensible point cloud could be produced. I use ubuntu18.04.3, pytorch 1.6, cuda10.1, tf_ops were compiled with gcc-4.8, using test.lmdb and checkpoint for rfa downloaded from google drive, environments all worked fine, but failed in evaluation.

**Comment by brandon9838** [comment]
Hi, it works fine for me.
Did you put lock lmdb in the same file？
![6601d179e754149e2f710dc8afceb40e'](https://user-images.githubusercontent.com/55751452/96867638-8cd42100-149f-11eb-8241-00a0277c8730.png)
![acc](https://user-images.githubusercontent.com/55751452/96867641-8e054e00-149f-11eb-8104-27035c346aa2.png)

**Comment by Leerw** [comment]
yes, I trained a new small model to validate my data and environments on test.lmdb and test.lmdb-lock, worked well. But evaluating on official pretrained weights failed.

**Comment by XLechter** [comment]
@Leerw 
I don't know if you reimplement the project in pytorch? as you said you use pytorch 1.6. I am not sure if the checkpoint can be shared well between pytorch and tensorflow, and the implementation details in your project. But if your self-trained model can work, you can try to train more epochs to get similar results.

Another possible issue I can think about is in the 41 line in eval.py 'model = model_module.Model(inputs, gt, theta, False)'. If you set the last parameter to False? This is a parameter to judge if the model is in training, since the bn operation is different in training and testing.

**Comment by XLechter** [comment]
Hi @brandon9838 ,thanks a lot for your comment! I close this issue as I don't want to bother you more 😂 .  @Leerw  can email me or create a new issue if you try on another computer or new environment and still face the same problems, thank you.
